### PR TITLE
AWS Resources

### PR DESCRIPTION
**Pull request by redpandarevolt**
Added Markup content for video links and their descriptions.
 - Behold I have created a Pull Request!

### CODE REVIEW

File: _docs/aws_resources.md [hunk 1]
```diff
@@ -25,8 +25,40 @@ After you have signed up for an AWS account and begin using services, keep in mi
...
+The list above should be sufficient to get you started. You **do not** have to follow every step. However, pay close attention to **1, 3, and 5** because those are some of the most common solutions.
 
+For additional information, AWS has a [comprehensive guide](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/checklistforunwantedcharges.html) to help you avoid unexpected charges if you find yourself in an uncharted territory.
+
+
+
+# Video Tutorials To Help You Get Familiar With AWS Cloud services
+
+## Static Website Hosting on S3, Route 53 Domain Name Registration
+In this AWS S3 tutorial the presenter walks step-by-step through the process of hosting a website on Amazon S3. Then set up a custom domain name in AWS Route 53 and associate it with our website in S3
```
Comment (etcadinfinitum) [inline _docs/aws_resources.md]:
```suggestion
In this AWS S3 tutorial the presenter walks step-by-step through the process of hosting a website on Amazon S3. The tutorial also includes setting up a custom domain name in AWS Route 53 and associating it with the website in S3.
```

File: _docs/aws_resources.md [hunk 2]
```diff
@@ -25,8 +25,40 @@ After you have signed up for an AWS account and begin using services, keep in mi
...
+(Domain registration is not free but at an affordable price. )
```
Comment (etcadinfinitum) [inline _docs/aws_resources.md]: Use an extra line break before this line to keep this in a separate paragraph if you like; I have not added an extra line. I personally like 
```suggestion
(Note: domain registration is not free but many custom domains are available at an affordable price.)
```

File: _docs/aws_resources.md [hunk 3]
```diff
@@ -25,8 +25,40 @@ After you have signed up for an AWS account and begin using services, keep in mi
...
+
+[S3 Video Tutorial Available Here ] (https://www.youtube.com/watch?v=P_B79xXf-w8)
```
Comment (etcadinfinitum) [inline _docs/aws_resources.md]: Link syntax is very, very particular; can't have spaces between the `]` bracket and `(` paren. Take a look at the [rendered markdown](https://github.com/UWB-ACM/Hackathon-Docs-2020/blob/490cd5ec451855cfa37d06f50d5bf9cb4161e5c3/_docs/aws_resources.md) to see what I'm talking about.

```suggestion
[S3 Video Tutorial Available Here](https://www.youtube.com/watch?v=P_B79xXf-w8)
```

File: _docs/aws_resources.md [hunk 4]
```diff
@@ -25,8 +25,40 @@ After you have signed up for an AWS account and begin using services, keep in mi
...
+
+## AWS Lambda & DynamoDB | AWS Serverless Tutorial
+This tutorial as uses AWS Lambda, AWS API Gateway and DynamoDB to create a serverless backend for your application. This is a three
+part series that can be accessed on "Cloud Path" via youTube.
+
+[Lambda DynamoDB Video Tutorial Available Here] (https://www.youtube.com/watch?v=VGerk8hrP9U)
```
Comment (etcadinfinitum) [inline _docs/aws_resources.md]: Link syntax is very, very particular; can't have spaces between the `]` bracket and `(` paren. Take a look at the [rendered markdown](https://github.com/UWB-ACM/Hackathon-Docs-2020/blob/490cd5ec451855cfa37d06f50d5bf9cb4161e5c3/_docs/aws_resources.md) to see what I'm talking about.

Also, here, I like bullet points for the two videos. Using bullets ensures that your two lines get respected with line breaks; [line breaks in GitHub markdown have some tricks to it.](https://gist.github.com/shaunlebron/746476e6e7a4d698b373)

```suggestion
* [Lambda DynamoDB Video Tutorial Available Here](https://www.youtube.com/watch?v=VGerk8hrP9U)
```

File: _docs/aws_resources.md [hunk 5]
```diff
@@ -25,8 +25,40 @@ After you have signed up for an AWS account and begin using services, keep in mi
...
+[Reserved Words In DynamoDB] (https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/ReservedWords.html)
```
Comment (etcadinfinitum) [inline _docs/aws_resources.md]: Link syntax is very, very particular; can't have spaces between the `]` bracket and `(` paren. Take a look at the [rendered markdown](https://github.com/UWB-ACM/Hackathon-Docs-2020/blob/490cd5ec451855cfa37d06f50d5bf9cb4161e5c3/_docs/aws_resources.md) to see what I'm talking about.

Also, here, I like bullet points for the two videos. Using bullets ensures that your two lines get respected with line breaks; [line breaks in GitHub markdown have some tricks to it.](https://gist.github.com/shaunlebron/746476e6e7a4d698b373)

```suggestion
* [Reserved Words In DynamoDB](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/ReservedWords.html)
```

File: _docs/aws_resources.md [hunk 6]
```diff
@@ -25,8 +25,40 @@ After you have signed up for an AWS account and begin using services, keep in mi
...
+## How to Create EC2 Instance in AWS Step-by-step
+Create Your EC2 Resources and Launch Your EC2 Instance.
+How to create EC2 instance in AWS step by step.
+
+[EC2 Video TutorialAvailable here](https://www.youtube.com/watch?v=1x5dWRaLvIw)
+
+## AWS EC2 Linux Launch & Connection Using SSH
+
+[EC2 Linux Launch Video Available Here] (https://www.youtube.com/watch?v=v0g1M5bb9u4)
+[EC2 connect with Putty(Windows)] (This video will show how to use a PuTTY private key to connect to your Amazon EC2 Linux instance.)
```
Comment (etcadinfinitum) [inline _docs/aws_resources.md]: Reorganize this paragraph to align with the sections above.

```suggestion
This video will show how to use a PuTTY private key to connect to your Amazon EC2 Linux instance.

* [EC2 Linux Launch Video Available Here](https://www.youtube.com/watch?v=v0g1M5bb9u4)
```

File: _docs/aws_resources.md [hunk 7]
```diff
@@ -25,8 +25,40 @@ After you have signed up for an AWS account and begin using services, keep in mi
...
+## AWS Lamda Tutorial | Use AWS CloudWatch Events to Schedule a Lambda Function Invocation
+In this AWS Lambda tutorial we set up a basic hello world Lambda function that's triggered by a scheduled CloudWatch event to run every minute. This video is part of the AWS Serverless tutorial series.
+
+[Lambda & CloudWatch Video Tutorial Available Here.](https://www.youtube.com/watch?v=-v4LMV5DAD4&list=PLD_RqipW0-9s-u1HXTglYV8Aam-5P3XLi&index=7&t=0s)
+
+## How to Create EC2 Instance in AWS Step-by-step
+Create Your EC2 Resources and Launch Your EC2 Instance.
+How to create EC2 instance in AWS step by step.
+
+[EC2 Video TutorialAvailable here](https://www.youtube.com/watch?v=1x5dWRaLvIw)
```
Comment (etcadinfinitum) [inline _docs/aws_resources.md]: Rephrase & spaces

```suggestion
How to create EC2 instance in AWS step by step, including creating the EC2 resource and launching the instance.

[EC2 Video Tutorial Available Here](https://www.youtube.com/watch?v=1x5dWRaLvIw)
```

**Review by etcadinfinitum — APPROVED** [review]
Overall, this looks fine. Approving on the condition that all suggestions are applied. 

To add the suggestion, under each box, click "Commit suggestion" and feel free to accept the default commit message.

File: _docs/aws_resources.md (hunk 6 above)
Comment (redpandarevolt) [inline _docs/aws_resources.md]: [EC2 connect with Putty(Windows)] (https://www.youtube.com/watch?v=bi7ow5NGC-U)
     This video will show how to use a PuTTY private key to connect to your Amazon EC2 Linux instance.

**Pull request merged**